From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:16 +0800
Subject: fo_FO-date_fmt

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix d_t_fmt and date_fmt in fo_FO
# DP: Related bugs: #307194
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Jacob Sparre Andersen <sparre@nbi.dk>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix d_t_fmt and date_fmt in fo_FO
# DP: Related bugs: #307194
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Jacob Sparre Andersen <sparre@nbi.dk>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10
---
 localedata/locales/fo_FO | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/localedata/locales/fo_FO b/localedata/locales/fo_FO
index f1f7872..3e9816e 100644
--- a/localedata/locales/fo_FO
+++ b/localedata/locales/fo_FO
@@ -98,8 +98,8 @@ mon     "januar";/
         "oktober";/
         "november";/
         "desember"
-d_t_fmt  "%a %d %b %Y %T"
-date_fmt "%a %d %b %Y %T %Z"
+d_t_fmt "%a. %d. %b. %Y %T %Z"
+date_fmt "%1 tann %-e. %B %Y klokkan %T (UTC%z)"
 d_fmt   "%d//%m-%Y"
 t_fmt   "%T"
 am_pm   "";""
